hedera-node/src/test/java/com/hedera/services/state/merkle/MerkleUniqueTokenSerdeTest.java

```diff
@@ -21,26 +21,39 @@
  */
 
 import com.hedera.services.state.submerkle.EntityId;
+import com.hedera.services.utils.NftNumPair;
 import com.hedera.test.serde.SelfSerializableDataTest;
 import com.hedera.test.utils.SeededPropertySource;
 
+import static com.hedera.services.state.merkle.MerkleUniqueToken.RELEASE_0180_VERSION;
+import static com.hedera.services.state.merkle.MerkleUniqueToken.RELEASE_0250_VERSION;
+
 public class MerkleUniqueTokenSerdeTest extends SelfSerializableDataTest<MerkleUniqueToken> {
+	public static final int NUM_TEST_CASES = 2 * MIN_TEST_CASES_PER_VERSION;
+
 	@Override
 	protected Class<MerkleUniqueToken> getType() {
 		return MerkleUniqueToken.class;
 	}
 
 	@Override
 	protected MerkleUniqueToken getExpectedObject(final SeededPropertySource propertySource) {
-		return propertySource.nextMerkleUniqueToken();
+		return propertySource.next0260UniqueToken();
 	}
 
 	@Override
 	protected MerkleUniqueToken getExpectedObject(final int version, final int testCaseNo) {
-		var expected = super.getExpectedObject(version, testCaseNo);
-		if (version < MerkleUniqueToken.RELEASE_0250_VERSION) {
-			expected.setSpender(EntityId.MISSING_ENTITY_ID);
+		final var propertySource = SeededPropertySource.forSerdeTest(version, testCaseNo);
+		final var seededObject = getExpectedObject(propertySource);
+		if (version <= RELEASE_0180_VERSION) {
```

Comment:
Shouldn't this be < RELEASE_0250_VERSION

Reply:
would be same right.

Reply:
It will fail for any version that is between 0.18.0 - 0.25.0 right ? Since it is deserialized  for all versions < 0.25.0

Reply:
These versions are internal to MerkleUniqueToken, 
```
        static final int RELEASE_0180_VERSION = 2;
	static final int RELEASE_0250_VERSION = 3;
	static final int RELEASE_0260_VERSION = 4;
```

so there are no other versions in-between